.gitignore

```diff
@@ -62,7 +62,6 @@ build/
 *.rar
 *.err
 
-temp/
```

Comment:
why you removed this one?

Reply:
Oh...I see...it is kind of wrong to do this like that...maybe keep temp/ here and rename temp/tests to work.in.progress.tests?

Reply:
@Fishgame1 I was thinking about creating a new task for the basic tests rework - to be discussed tomorrow.